schema.md

```diff
@@ -0,0 +1,555 @@
+
+### Key `description/platforms`
+
+* Type: enumerated string or array of strings
+* Presence: mandatory
+* Values: "web", "windows", "mac", "linux", "ios", "android". Values
+  outside this list are allowed.
+* Example: "web"
+
+This key specifies which platform the software runs on. It is meant
+to describe the platforms that users will use to access and operate
+the software, rather than the platform the software itself runs on.
+
+Use the predefined values if possible. If the software runs on a
+platform for which a predefined value is not available, a different
+value can be used.
+
+### Key `description/logo`
+
+* Type: string or array of strings (path to file)
+* Presence: optional
+* Formats: SVG, PNG
+* Example: "img/logo.svg"
+
+This key contains the logo of the software. Logos should be in vector
+format, but raster formats are allowed as a fallback.
+
+Vector logos should be provided at least in two formats: a color one,
+and a monochromatic (black) one.
+
+Raster logos should be provided only if vector logos do not exist.
+In this case, they should be transparent PNGs, minimum 1000px of
+width.
+
+### Key `description/shortdesc`
+
+* Type: multi-language string (max 100 chars)
+* Presence: mandatory
+* Example: "Advanced booking system for hospitals"
+
+This key contains a short description of the software. It should be
+a single line containing a single sentence. Maximum 100 characters are
+allowed.
+
+### Key `description/longdesc`
+
+* Type: multi-language string (min 500 chars, max 10000 chars)
+* Presence: mandatory
+
+This key contains a longer description of the software, between 500
+and 10000 chars. It is meant to provide an overview of the capabilities
+of the software for a potential users. The audience for this text
+should be that of users of the software, not developers. You can think
+of this text as the description of the software that would be in its
+website (if the software had one).
+
+### Key `description/screenshots`
+
+* Type: array of strings (paths)
+* Presence: optional
+* Formats: PNG, JPG
+* Example: "data/screenshots/configuration.png"
+
+This key contains one or multiple paths to files showing screenshots of
+the software. They are meant to give a quick idea on how the software
+looks like and how it works.
+
+Screenshots can be of any shape and size; the suggested formats are:
+
+ * Desktop: 1280x800 @1x
+ * Tablet: 1024x768 @2x
+ * Mobile: 375x667 @2x
+
+
+### Key `description/videos`
+
+* Type: array of strings (URLs)
+* Presence: optional
+* Example: "https://youtube.com/xxxxxxxx"
+
+This key contains one or multiple URLs of videos showing how the
+software works. Like screenshots, videos should be used to give a quick
+overview on how the software looks like and how it works. Videos should
+be hosted on a video sharing website that supports the
+[oEmbed](https://oembed.com) standard; popular options are YouTube and
+Vimeo.
+
+
+
+
+
+
+
+
+
+
+
+## Section `meta`
```

Comment:
I'm wondering if `meta` might be the wrong word here. Since the description etc is also meta-data, and maybe even everything in this file.

May I suggest the word `context`?

Reply:
OK.